spec/flipper/engine_spec.rb

```diff
@@ -106,6 +108,82 @@
     end
   end
 
+  context 'with cloud' do
+    let(:env) do
+      { "FLIPPER_CLOUD_TOKEN" => "test-token" }
+    end
+
+    # App for Rack::Test
+    let(:app) { application.routes }
+
+    it "initializes cloud configuration" do
+      stub_request(:get, /flippercloud\.io/).to_return(status: 200, body: "{}")
+
+      ENV.update(env)
```

Comment:
Does this need to be rolled back after spec is done?

Reply:
I've been using https://github.com/thoughtbot/climate_control for modifying env of late. Its nice.

Reply:
Good call. I added climate_control and updated everywhere else that we were modifying ENV in 4e4decb